SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -28,7 +28,7 @@ BuildRequires:  fuse-devel
 # kernel-uvm is required for debuggability, exercising confidential guest (confidential_guest=true)
 # code paths without actual SEV SNP enablement (sev_snp_guest=false)
 Requires:  kernel-uvm
-Requires:  moby-containerd-cc
+Requires:  containerd
```

Comment:
containerd2?

Reply:
I think both methods work here. Since you both will be part of the primary maintainer crew for this package, choose the method that will cause the least confusion during future Azure Linux 3.0 maintenance.